<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendRequest</class>
 <widget class="QDialog" name="friendRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
		background-color:  qlineargradient(
		spread:pad, x1:0, y1:0 x2:0, y2:0,
		stop: 0 #A3D8F4, stop:1 #E6F7FF
		);
		border: 2px solid #ffffff;
		boder-radius: 10px;
}

QLineEdit {
		border: 2px solid #E63946;
		border-radius: 5px;
		padding: 5px;
		font-size: 14px;
		background-color: #ffffff;
		color: #333333;
}

QLineEdit:forcus {
		border: 2px solid #457B9D;
		background-color: #F1FAEE;
}

QPushButton{
		background-color: #2A9D8F;
		color: white;
		border: none;
		border-radius: 15px;
		padding: 10px 20px;
		font-size: 14px;
}

QPushButton:hover{
background-color: #264653;
}

QPushButton:pressed{
background-color: #1B3B3A;
}

QLabel{
font-size :14px;
color: #1D3557;
font-weight: bold;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Send request  </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/request-send.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Accept  </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/accept.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Online Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/refresh_icon.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>List Request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Decline </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/decline.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/refresh_icon.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Back  </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/return.png</normalon>
    </iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
